doc/source/dev/roadmap-detailed.rst

```diff
@@ -234,27 +234,24 @@ Ideas for new features:
 
 - Add type-generic wrappers in the Cython BLAS and LAPACK
 - Make many of the linear algebra routines into gufuncs
+- Complete support for batched operations (see
+  `gh-21466 <https://github.com/scipy/scipy/issues/21466>`__)
```

Comment:
We basically already have complete support for batched operations now! The remaining task is, where appropriate, move the batching to low level code

Reply:
I still saw heavy activity on this topic, so it made sense to me to include it - and the optimizations seem necessary before declaring victory? Happy to leave it out if that seems better of course. @ev-br any preference?

Reply:
IMO it's good to have this is in the roadmap, thanks Ralf for adding it.

While it's true that in 1.16 we'll have batching across scipy.linalg (yay), there is a ~5x--10x performance differential on the table, and linear algebra is low enough in the stack to make it crucial, not "just some perf optimizations".
And ongoing work around this topic will result in a very clear message of what is the difference between numpy.linalg and scipy.linalg. If anything, we could actually expand the scipy.linalg roadmap but that definitely can be done in a sequel. The current wording LGTM.